### PR TITLE
Fix missing resource details

### LINKED ISSUE #38: Missing resource details
In the original system, resources have a creation date and an extent field, see http://web.archive.org/web/20130327065029/http://www.digitalisiertedrucke.de/record/491272?ln=de

In our current data, these are not present:
http://test.digitalisiertedrucke.de/resources/D491272?format=json
http://test.digitalisiertedrucke.de/resources/P491272?format=json

Publisher field looks incomplete too ('Universitäts- und Landesbibliothek').

### PR DESCRIPTION
**Pull request by fsteeg**
Will resolve #38

### CODE REVIEW

File: conf/TransformationZvdd-title-print.xml [hunk 1]
```diff
@@ -52,7 +52,7 @@
 						<cgxml:literal name="medium" value="http://rdaregistry.info/termList/RDAproductionMethod/1010"/>
 						<cgxml:literal name="fulltextOnline" value="http://digi.ub.uni-heidelberg.de/diglit/falke1873"/>
 						<cgxml:literal name="title" value="Kunst und Kunstgewerbe auf der Wiener Weltausstellung 1873"/>
-						<cgxml:literal name="issued" value="1873"/>
+						<cgxml:literal name="issued" value="07.04.1873[nach dem 07.04.1873]"/>
```
Comment (dr0i) [inline conf/TransformationZvdd-title-print.xml]: not harmonising the values will result in not so useful facets and you can't do all kinds of sorting and ranging, see e.g. https://github.com/hbz/lobid/pull/46.
I don't know what is more important, though. Best would be two have two fields: `issued` and maybe `issued-notes`.

Comment (acka47) [inline conf/TransformationZvdd-title-print.xml]: If the "Zeitbezug" facet at http://beta.digitalisiertedrucke.de/ is built on this data, we shouldn't change it. If the facet relies on another field (which?), I don't see a problem with this adjustment.

Comment (dr0i) [inline conf/TransformationZvdd-title-print.xml]: The problem would be to have lots (?) of facets with only one hit and you wouldn't hit this one when facetting on `1873`.

Comment (fsteeg) [inline conf/TransformationZvdd-title-print.xml]: I've restored the previous behavior, see 29b754dd61a69f10fec7c860e87fe09562d73aab.

**Comment by dr0i** [comment]
+1

**Pull request merged**